definitions/divorce/json/CaseEventToFields.json

```diff
@@ -1670,15 +1672,156 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorCreate",
-    "CaseFieldID": "SeparationLivedTogetherAsCoupleAgainDetails",
+    "CaseFieldID": "D8PhysicalSeparationDate",
     "PageFieldDisplayOrder": 4,
     "DisplayContext": "MANDATORY",
     "PageID": "SolSOCSeparation",
     "PageLabel": "Statement of case - separation",
     "PageDisplayOrder": 13,
-    "FieldShowCondition": "SeparationLivedTogetherAsCoupleAgain=\"Yes\"",
     "ShowSummaryChangeOption": "Yes"
   },
+  {
+    "LiveFrom": "05/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "D8SeparationReferenceDate",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "SolSOCLivedApart",
+    "PageLabel": "Lived Apart for the entire time",
+    "PageDisplayOrder": 14,
+    "PageShowCondition": "D8ReasonForDivorce=\"separation-*\"",
```

Comment:
The pageshow condition will be updated to D8ReasonForDivorce="separation-*" OR D8ReasonForDivorce="desertion" once RDM-2804 is resolved